test/move_to_front_test.cpp

```diff
@@ -128,13 +123,15 @@ TEST(MoveToFront, InsertLeftRotation) {
 
   CheckTree(mtf, std::string(R"(
 30H2S2----20H1S1----D2
-)").substr(1));
+)")
+                     .substr(1));
```

Comment:
This doesn't look right.

Reply:
This one is borderline for me.  There's too many lines that would need to be disabled and it's easier to just let clang-format decide here.

Reply:
I agree it does not look good, but it is in an part of the code that people are going to read very often.  Since it happens in a few different file in a lot of places, I don't think it is worth fixing until some needs to actually change those files.  I suggest just letting clang-format do its thing.